samples/csharp/MicrosoftEduGraphSamples/Workflows/AssignmentWorkflow.cs

```diff
@@ -34,7 +34,7 @@ public async Task<IEnumerable<EducationAssignment>> GetMeAssignmentsFromNonArchi
                 string userAccount = isTeacher ? _config["teacherAccount"] : _config["studentAccount"];
 
                 // Get a Graph client using delegated permissions
-                var graphClient = GraphClient.GetDelegateClient(_config["tenantId"], _config["appId"], userAccount, _config["password"]);
+                var graphClient = MicrosoftGraphSDK.GraphClient.GetDelegateClient(_config["tenantId"], _config["appId"], userAccount, _config["password"]);
```

Comment:
REVERT, already covered in https://github.com/microsoft/edu-assignments-graph-sdk/pull/16/files

Reply:
reverted the changes

Reply:
you didn't revert it, apply the suggestion

```suggestion
                var graphClient = GraphClient.GetDelegateClient(_config["tenantId"], _config["appId"], userAccount, _config["password"]);
```